#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Outputs a NameError exception and a input message."""
    raise NameError(message)
